descriptor.md

```diff
@@ -161,6 +162,16 @@ Implementations MAY implement SHA-512 digest verification for use in descriptors
 When the _algorithm identifier_ is `sha512`, the _encoded_ portion MUST match `/[a-f0-9]{128}/`.
 Note that `[A-F]` MUST NOT be used here.
 
+#### BLAKE3
```

Comment:
In addition to the new section, you'll want a new entry in [the registered-algorithms table][1] up around line 140, right?

[1]: https://github.com/opencontainers/image-spec/blob/fbb4662eb53b80bd38f7597406cf1211317768f0/descriptor.md?plain=1#L141-L144

Reply:
Updated